server/client.go

```diff
@@ -73,6 +74,10 @@ type IssueService interface {
 	UpdateComment(issueKey string, comment *jira.Comment) (*jira.Comment, error)
 }
 
+type LabelService interface {
+	GetLabels(value, url string) (LabelResult, error)
```

Comment:
Maybe rename to `SearchLabels`?

Reply:
I renamed this to something more generic to be used for autocomplete searches of other types of fields